lib/game.rb

```diff
@@ -0,0 +1,90 @@
+class Game
+  attr_accessor :player_1, :player_2, :board
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+  def initialize(player_1 = Players::Human.new('X'), player_2 = Players::Human.new('O'), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    return @player_1 if board.turn_count.even?
+    return @player_2 if board.turn_count.odd?
+  end
+
+  def won?
+    WIN_COMBINATIONS.each do |win_combo|
+      if board.cells[win_combo[0]] == 'X' && board.cells[win_combo[1]] == 'X' && board.cells[win_combo[2]] == 'X'
+        return true
+      elsif board.cells[win_combo[0]] == 'O' && board.cells[win_combo[1]] == 'O' && board.cells[win_combo[2]] == 'O'
+        return true
+      end
+    end
+    return false
+  end
+
+  def draw?
+    !won? && board.full?
+  end
+
+  def over?
+    won? || draw?
+  end
+
+  def winner
+    WIN_COMBINATIONS.each do |win_combo|
```

Comment:
Consider using your `won?` method here instead of iterating through the WIN_COMBINATIONS array again. 

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable to return the token of the winning player.